backend/src/handlers/player.handler.ts

```diff
@@ -86,7 +86,7 @@ export const playerJoin = async (io: Server, socket: Socket): Promise<void> => {
     const player = await getPlayer(game.gameCode, playerId, game);
     if (player.new) {
       await initialisePlayer(game, playerId);
-      socket.to(game.gameCode).emit("players:add", player.nickname);
+      socket.to(game.gameCode).emit("players:add", player.id, player.nickname);
     }
     socket.data.nickname = player.nickname;
```

Comment:
What does this do? Do you need to add one for id?

Reply:
This returns the args `player.id` and `player.nickname` for the event `players:add`. This corresponds to the frontend implementing the following:

`socket.on("players:add", (id, nickname) => { });`

Reply:
I was referring to ``socket.data.nickname = player.nickname;``

Reply:
Ah gotcha - if you take a look at `auth.handler.ts` `socket.data.gameCode` & `socket.data.playerId` are already set 👌🏼